pkgs/applications/misc/bloodhound/default.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gnome2
+, gtk3
+, libGL
+, libappindicator-gtk3
+, libdrm
+, libnotify
+, libpulseaudio
+, libuuid
+, libxcb
+, libxkbcommon
+, libxshmfence
+, mesa
+, nspr
+, nss
+, pango
+, systemd
+, unzip
+, xdg-utils
+, xorg
+}:
+stdenv.mkDerivation rec {
+  pname = "bloodhound";
+  binaryName = "BloodHound";
+  version = "4.3.1";
+  src = fetchurl {
+    url = "https://github.com/BloodHoundAD/BloodHound/releases/download/v${version}/BloodHound-linux-x64.zip";
+    sha256 = "sha256-OtQNrbPGAw5WoDeli6+REcx9ajor0CeF89L7Gg44aB0=";
+  };
+
+  rpath = lib.makeLibraryPath [
+    alsa-lib
+    at-spi2-atk
+    at-spi2-core
+    atk
+    cairo
+    cups
+    dbus
+    expat
+    fontconfig
+    freetype
+    gdk-pixbuf
+    glib
+    gnome2.GConf
+    gtk3
+    libGL
+    libappindicator-gtk3
+    libdrm
+    libnotify
+    libpulseaudio
+    libuuid
+    libxcb
+    libxkbcommon
+    mesa
+    nspr
+    nss
+    pango
+    systemd
+    xorg.libX11
+    xorg.libXScrnSaver
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXtst
+    xorg.libxkbfile
+    xorg.libxshmfence
+  ] + ":${stdenv.cc.cc.lib}/lib64";
+
+  buildInputs = [
+    gtk3 # needed for GSETTINGS_SCHEMAS_PATH
+  ];
+
+  nativeBuildInputs = [ makeWrapper unzip ];
+
+  dontUnpack = true;
+  dontBuild = true;
+  dontPatchELF = true;
```

Comment:
Why not?

Reply:
Why not about all three booleans ? or just `dontPatchELF` ? 
- dontBuild : the source here is already prebuilt as a binary
- dontUnpack : I'm not sure, what this options does exactly

I didn't make these choices, but the previous PR, was done so that way maybe because it'd be easier, I can understand that this isn't the proper way to do it, and can give a try on making this more according to what is standard/recommanded.

Reply:
The phases should not be disabled unless they cause issues. If buildPhase is a no-op then leaving it enabled is preferred. unpackPhase/dontUnpack unpacks the downloaded source from src.